src/RemoteTech/RTLog.cs

```diff
@@ -5,46 +5,119 @@ namespace RemoteTech
 {
     public static class RTLog
     {
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <value></value>
         private static readonly bool verboseLogging;
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <value></value>
+        public static int maxDebugLevels = 7;
```

Comment:
No. I'll change the current debuglvl-concept with enums. So maybe we don't need this member anymore.